jbmc/regression/jbmc-strings/ConstantEvaluationToUpperCase/pi.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+Main.class
+--function Main.pi --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^Generated [1-9]\d* VCC\(s\), [1-9]\d* remaining after simplification$
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Surprising! Pi *should* be affected by the real JDK method; is our models lib wrong here?

Reply:
the builtin upper_case and lower_case are only correct for a (small) subset of unicode (I think ascii and latin-1 supplement) so this is incorrect but expected. It sould be marked as knownbug.